src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -130,7 +130,7 @@ public void setServerUrl(String serverUrl) {
   }
 
   public Map<String, Object> getDictionary() {
-    return dictionary;
+    return new HashMap<>(dictionary);
```

Comment:
can we instead do:
```
   public void setDictionary(Map<String, Object> dictionary) {
-    this.dictionary = dictionary;
+    this.dictionary = Collections.unmodifiableMap(dictionary);
   }
```

Reply:
Indeed, this way we do not have to create new copies each time the dictionary is requested.